hack/versions-management/tag_reconciler.sh

```
@@ -11,4 +11,4 @@ if [ "${DRY_RUN:-false}" = "true" ]; then
   ARGS+=(--dry-run)
 fi
 
-python "$SCRIPT_DIR/tag_reconciler.py" "$@"
+python "$SCRIPT_DIR/tag_reconciler.py" "${ARGS[@]}
```

Shellcheck (0.10.0)

[error] Couldn't parse this double quoted string. Fix to allow more checks.

(SC1073)